testplans/lotus-soup/_compositions/baseline-k8s-1-1.toml

```diff
@@ -0,0 +1,67 @@
+[metadata]
```

Comment:
I am bundling a few more composition files for easier debugging, when we notice that tests are broken / flaky.